### PR TITLE
[Issue-160] Option to set a Header

### PR DESCRIPTION
**Pull request by montekaka**
[Issue-160](https://github.com/honeybadger-io/heya/issues/160)

## Details
Added a `:headers` option to Heya campaigns

### CODE REVIEW

**Review by joshuap — COMMENTED** [review]
This looks good to me, thanks @montekaka!

**Review by joshuap — APPROVED** [review]

File: app/mailers/heya/campaign_mailer.rb [hunk 1]
```diff
@@ -23,6 +23,12 @@ def build
       instance_variable_set(:"@#{user.model_name.element}", user)
       instance_variable_set(:@campaign_name, campaign_name)
 
+      headers_params = step.params.fetch("headers", nil)
+
+      if headers_params
+        headers headers_params
+      end
+
```
Comment (joshuap) [inline app/mailers/heya/campaign_mailer.rb:31]:
```suggestion
      headers_params = step.params.fetch("headers", nil)
      headers(headers_params) if headers_params
```

**Review by joshuap — APPROVED** [review]
One tiny suggestion to make it more compact.

File: app/mailers/heya/campaign_mailer.rb (hunk 1 above)
Comment (montekaka) [reply app/mailers/heya/campaign_mailer.rb]: Thank you so much for the suggestion!  changed!

**Pull request merged**